hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+This patch fixes invalid annotations detected for the tests generated by
+:doc:`Ghostwritter <ghostwriter>`. It will now correctly generate `Optional`
+types with just one type argument and handle union expressions inside of type
+arguments correctly. Additionally, it now supports code with the
+`from __future__ import annotations` marker for Python 3.10 and newer.
```

Comment:
```suggestion
:doc:`Ghostwritter <ghostwriter>`. It will now correctly generate ``Optional``
types with just one type argument and handle union expressions inside of type
arguments correctly. Additionally, it now supports code with the
``from __future__ import annotations`` marker for Python 3.10 and newer.
```

While single-backticks mean `monospace text` in markdown, for .rst files you need double-backticks (and single is variously some kind of cross-reference, or italics).  [Here's a preview of the rendered docs](https://hypothesis--3556.org.readthedocs.build/en/3556/changes.html#current-pull-request), via our PR build.